src/main/java/com/ajou/hertz/domain/user/service/UserCommandService.java

```diff
@@ -131,6 +135,31 @@ private String generateRandom16CharString() {
 	}
 
 	/**
+	 * 유저의 프로필 이미지를 업데이트한다.
+	 *
+	 * @param userId 유저 id
+	 * @param newProfileImage 새로운 프로필 이미지
+   *
+	 * @return 업데이트된 유저 정보
+	 */
```

Comment:
```suggestion
	/**
	 * 유저의 프로필 이미지를 업데이트한다.
	 *
	 * @param userId 유저 id
	 * @param newProfileImage 새로운 프로필 이미지
	 * @return 업데이트된 유저 정보
	 */
```

혼자 튀어나와 있는 친구 ㅎㅎ 정리해주세요 :)